llvm/lib/Target/AIE/AIE2MultiSlotPseudoInstrInfo.td

```diff
@@ -19,11 +19,11 @@
 let mayLoad = false, mayStore = false, hasSideEffects = false, Itinerary = II_PADD in {
   let Constraints = "$ptr = $res" in {
     def PADD_mod_pseudo : MultiSlot_Pseudo< (outs eP:$res), (ins eP:$ptr, eM:$mod),
-                          "padd_mod_pseudo", "[$ptr], $mod", 
-                          [PADDB_ldb_ptr_inc_nospill_nrm, PADDA_lda_ptr_inc_idx] >;
+                          "padd_mod_pseudo", "[$ptr], $mod", [PADDB_ldb_ptr_inc_nospill_nrm, PADDS_st_ptr_inc_idx, PADDA_lda_ptr_inc_idx] >;
     def PADD_imm_pseudo : MultiSlot_Pseudo< (outs eP:$res), (ins eP:$ptr, imm9x4:$imm),
```

Comment:
To be consistent with the naming of multi-slot loads, maybe change to PADD_imm10_pseudo and PADD_imm9_pseudo?